src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -1264,7 +1264,7 @@ public boolean loadFromUrl(URL url, FetchFromUrlMode mode) {
         }
     }
 
-    private boolean performLoadFromUrl(URL url, FetchFromUrlMode mode) {
+    private boolean performLoadFromUrl(URI url, FetchFromUrlMode mode) {
```

Comment:
Optional: should method signatures like this be changed?
`fromURL` -> `fromURI`

Reply:
Its still only a Url inside a URI object, as we check for protocols http & https